migen/build/platforms/sinara/humpback.py

```diff
@@ -0,0 +1,114 @@
+from migen.build.generic_platform import *
+from migen.build.lattice import LatticePlatform
+
+
+_io = [
+       ("clk25", 0, Pins("K9"), IOStandard("LVCMOS33")),
+
+       ("user_led", 0, Pins("H3"), IOStandard("LVCMOS33")),
+
+       ("serial0", 0,
+        Subsignal("rx", Pins("M13")),
+        Subsignal("tx", Pins("T11"), Misc("PULLUP")),
+        Subsignal("cts", Pins("T10"), Misc("PULLUP")),
+        Subsignal("rts", Pins("M15"), Misc("PULLUP")),
+        IOStandard("LVCMOS33"),
+        ),
+
+       ("serial1", 0,
+        Subsignal("rx", Pins("M11")),
+        Subsignal("tx", Pins("T13"), Misc("PULLUP")),
+        Subsignal("cts", Pins("B16"), Misc("PULLUP")),
+        Subsignal("rts", Pins("A6"), Misc("PULLUP")),
+        IOStandard("LVCMOS33"),
+        ),
+
+       ("spi", 0,
+        Subsignal("cs_n", Pins("R2")),
+        Subsignal("miso", Pins("T2")),
+        Subsignal("mosi", Pins("N5")),
+        Subsignal("clk", Pins("C8")),
+        IOStandard("LVCMOS33"),
+        ),
+
+       ("i2c", 0,
+        Subsignal("sda", Pins("T16"), Misc("PULLUP")),
+        Subsignal("scl", Pins("M12"), Misc("PULLUP")),
+        IOStandard("LVCMOS33"),
+        ),
```

Comment:
Nitpicking but other platforms indent this differently.

Reply:
Moving the bracket back by one space like on other platforms breaks PEP8 compliance (E124). I can change it anyway if you prefer.